src/plugins/subsystems/drm/drm.sh

```diff
@@ -349,9 +372,65 @@ function get_supported_mode_per_connector()
   printf '%s\n' "$modes"
 }
 
+# Creates a binary EDID file, for each connected monitor, on the current folder.
+function get_edid()
+{
+  declare -a monitors
+
+  monitors=( $(ls -d /sys/class/drm/*/edid) )
+
+  say ""
+
+  if [ ${#monitors[@]} -lt 1 ]; then
+	  complain "Could not find any listed monitors!"
+  fi
+
+  for i in ${monitors[@]};
+  do
+          echo $i | awk -F"/" '{print "Wrote: " $5".bin"}'
```

Comment:
Please, do not use `echo`, also check:

https://kworkflow.org/content/codingstyle.html#avoid-short-options
https://kworkflow.org/content/codingstyle.html#use-printf-instead-of-echo